### PR TITLE
feat: custom dimension custom range

### PR DESCRIPTION
**Pull request by rephus**
Closes: #5200

### Description:

Screenshot from 2023-10-30 10-41-41

Screenshot from 2023-10-30 10-41-36

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/components/Explorer/CustomDimensionModal/index.tsx [hunk 1]
```diff
@@ -219,6 +236,160 @@ export const CustomDimensionModal = () => {
                         />
                     )}
 
+                    {form.values.binType === BinType.CUSTOM_RANGE && (
+                        <>
+                            <Text fw={500}>Range</Text>
+                            {form.values.binConfig.customRange.map(
```
Comment (rephus) [inline packages/frontend/src/components/Explorer/CustomDimensionModal/index.tsx:242]: This `custom range` form can be extracted into a separated component. I'll create a ticket for this to do this on a separate PR

**Comment by ZeRego** [comment]
The code looks good but Render env is down so I couldn't test. 
Are we going to have a separate PR to be able to order by a custom range bin?

**Comment by rephus** [comment]
>  Are we going to have a separate PR to be able to order by a custom range bin?

Yes, that will be done separately

**Comment by rephus** [comment]
Redeploying render

**Review by ZeRego — APPROVED** [review]

**Pull request merged**